frontend/src/app/testResults/TestResultDetailsModal.tsx

```diff
@@ -113,12 +113,10 @@ export const DetachedTestResultDetailsModal = ({ data, closeModal }: Props) => {
     results &&
     results.some((d) => d.disease.name !== "COVID-19");
   if (hasMultiplexResults) {
-    displayResult["fluAResult"] = results.filter(
-      (d) => d.disease.name === "Flu A"
-    )[0].testResult;
-    displayResult["fluBResult"] = results.filter(
-      (d) => d.disease.name === "Flu B"
-    )[0].testResult;
+    displayResult["fluAResult"] =
+      results.filter((d) => d.disease.name === "Flu A")[0].testResult || null;
+    displayResult["fluBResult"] =
+      results.filter((d) => d.disease.name === "Flu B")[0].testResult || null;
```

Comment:
Can you add optional chaining (AKA the "?" operator) in the [0].testResult parts to avoid null pointer exceptions at runtime?

Reply:
Yep, I can take care of this